src/components/SearchForm/SearchForm.js

```diff
@@ -15,66 +15,150 @@ class SearchForm extends Component {
     super(props)
     
     this.state = {
-      formLayout: 'vertical'
+      formLayout: 'vertical',
+      maxTime: '',
+      haveIngredients: [],
```

Comment:
as you see here - we have an old version of names for form fields. and they are a bit confusing too.
can you update them too?

Reply:
I havent used this antd till now so I tried not to break any anything by keeping the names same

Reply:
it's ok, this was one of the reasons to ask you for help. i also thing that digging into other person code is form you like a software engineer better. at this repo we with my prev intern Rahul(he got a internship in Canada), am I right @rahuldcoder? we decided to get rid of bootstrap and try some framework that is focused on React.